renv/activate.R

```diff
@@ -31,6 +31,14 @@ local({
     if (!is.null(override))
       return(override)
 
+    # if we're being run in a context where R_LIBS is already set,
```

Comment:
I don't really understand why when run `renv::retore()` this file is updated on my end to exactly what is here and then git flags that i have uncomitted changes. Not sure if you observed and or understand this either?

Reply:
Not entirely sure. Reading the documentation, we have this description:

> Restore a project's dependencies from a lockfile, as previously generated by [snapshot()](http://127.0.0.1:27111/help/library/renv/help/snapshot). renv::restore() compares packages recorded in the lockfile to the packages installed in the project library. Where there are differences it resolves them by installing the lockfile-recorded package into the project library. If clean = TRUE, restore() will additionally delete any packages in the project library that don't appear in the lockfile.

Maybe you just need to run `clean = TRUE` to ensure your local differences (extra packages) are removed.